src/main/java/com/onaio/steps/handler/actions/ImportHandler.java

```diff
@@ -69,10 +71,16 @@ public boolean shouldOpen(int menu_id) {
     @Override
     public boolean open() {
         String deviceId = KeyValueStoreFactory.instance(activity).getString(HH_PHONE_ID);
-        String filename = Environment.getExternalStorageDirectory()+"/"+Constants.APP_DIR+"/"+Constants.EXPORT_FILE_NAME+"_"+
+        String surveyId = KeyValueStoreFactory.instance(activity).getString(HH_SURVEY_ID);
```

Comment:
It could be added in the settings, after the import URL.. The tricky bit will be the UX (how to define how the import URL suffix should look like)
![screenshot_20161005-144301](https://cloud.githubusercontent.com/assets/2384176/19112036/4275c36a-8b0b-11e6-9860-cd33f7e5900e.png)